tests/UnifiedSpecTests/change-streams/change-streams-disambiguatedPaths.json

```diff
@@ -0,0 +1,251 @@
+{
+  "description": "disambiguatedPaths",
```

Comment:
@levon80999: It wasn't my intention to steal your assigned tickets, but the spec changes for [PHPLIB-914](https://jira.mongodb.org/browse/PHPLIB-914) and [PHPLIB-924](https://jira.mongodb.org/browse/PHPLIB-924) were included when I synced to the commit for [PHPLIB-912](https://jira.mongodb.org/browse/PHPLIB-912).